### PR TITLE
Upgrade to Akka 2.3.11 to fix akka/akka #17390 - blockOn requirement failure

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

**Comment by marcospereira** [comment]
Hum... maybe we can also close #4383 here.

File: framework/project/Dependencies.scala [hunk 1]
```diff
@@ -106,12 +106,10 @@ object Dependencies {
   val jodatime = "joda-time" % "joda-time" % "2.7"
   val jodaConvert = "org.joda" % "joda-convert" % "1.7"
 
-  private val akkaCoreDeps = Seq("akka-actor", "akka-slf4j").map("com.typesafe.akka" %% _ % "2.3.9")
-
   def runtime(scalaVersion: String) =
     slf4j ++
     Seq("logback-core", "logback-classic").map("ch.qos.logback" % _ % "1.1.3") ++
-    akkaCoreDeps ++
+    Seq("akka-actor", "akka-slf4j").map("com.typesafe.akka" %% _ % "2.3.9") ++
```
Comment (richdougherty) [inline framework/project/Dependencies.scala]: Is that meant to be `2.3.11` here? :)

**Comment by richdougherty** [comment]
I can fix #4383 after this PR is merged.

File: framework/project/Dependencies.scala (hunk 1 above)
Comment (benmccann) [inline framework/project/Dependencies.scala]: Yes, thank. Updated

**Comment by richdougherty** [comment]
Thanks @benmccann!

**Pull request merged**